src/nft/components/bag/BagFooter.tsx

```diff
@@ -48,6 +50,7 @@ interface BagFooterProps {
   bagStatus: BagStatus
   fetchAssets: () => void
   assetsAreInReview: boolean
+  eventProperties: Record<string, unknown>
```

Comment:
i think we would ideally type this, but if we're not doing that elsewhere then it's okay